### DISCUSSION TITLE
Add support for doorbell

### DISCUSSION THREAD

**Discussion by sonar98** [andrei-tatar/node-red-contrib-smartnora#97] [Ideas]
It would be nice if support is added for doorbells.
I have a doorbell "switch" in domoticz.

I would like to have the possibility to receive a notification on my Google Home.

**Comment by andrei-tatar** [comment]
This is something that is on the roadmap. The service does not yet support pushing notifications to google home and the object detection trait but it's something useful to have.

**Reply by normanr** [reply]
I'd be interested in this too. I'm considering replacing my (non-functioning) doorbell with a battery powered zigbee button and would need some way to notify folks when it's pressed.

**Reply by andrei-tatar** [reply]
support for pushing notifications already added. I guess it's just a matter of adding the missing trait and the node to node-red. I will try to get to it sometime in the coming days.

**Comment by andrei-tatar** [comment]
Doorbell support is added in version 1.13.0

@normanr @sonar98 

https://github.com/andrei-tatar/node-red-contrib-smartnora/tree/master/doc/nodes/doorbell

**Reply by patrik-malina** [reply]
Hmm, i tried importing your sample, enabled notification in google home app for doorbell and tested inject nodes, but no notification on mine google mini. does it notify onli on google hub maybe?

**Reply by andrei-tatar** [reply]
@patrik-malina did you assign the doorbell to the same home as your Google home mini device? (By default it's unassigned)

**Reply by patrik-malina** [reply]
Yes i did

**Reply by andrei-tatar** [reply]
@patrik-malina It's working for me with any of the inject nodes. These are the settings I use in google home:
![Screenshot_20220501-093833](https://user-images.githubusercontent.com/6473183/166135225-4a4353e2-84b3-4828-9635-a07a1fedf91b.jpg)

**Reply by andrei-tatar** [reply]
@patrik-malina do you get any errors in the node-red logs?

**Reply by normanr** [reply]
I got it work work nicely. Initially I couldn't figure out the right input payload, so I imported the sample flow and changed the config to mine. I had to manually enable `Voice notifications` for the device in the Home app (and move it into a room in my Home) to get it to work for me (and then do it again when I moved the device between flows - I assume it's internal id changed). I assume that under settings for the `Home Mini` device under `Notification & digital wellbeing`, `Notifications` must also be enabled in order for things to work.

**Reply by patrik-malina** [reply]
> I got it work work nicely. Initially I couldn't figure out the right input payload, so I imported the sample flow and changed the config to mine. I had to manually enable `Voice notifications` for the device in the Home app (and move it into a room in my Home) to get it to work for me (and then do it again when I moved the device between flows - I assume it's internal id changed). I assume that under settings for the `Home Mini` device under `Notification & digital wellbeing`, `Notifications` must also be enabled in order for things to work.

Got it working too. Need to be IN ROOM, then it works.

Now it announce on TV and google mini, which is little annoying tho.

**Reply by normanr** [reply]
oh, I see what you mean. `Do not disturb` (also under `Notification & digital wellbeing`) seems to work (to suppress notifications) for the Google Mini, maybe TV has it too?

**Reply by patrik-malina** [reply]
Sorry for late respond. Was on a business trip. TV has no DND function, so still getting announced on both devices. 
@andrei-tatar maybe is that way per design? To announce to all devices in home?!?
It would be good if some could choose device to announce.

**Comment by alonsomarce** [comment]
Hi guys, Smart NORA could not be possible to add from the Google Home app. Are there any restriction on that?

**Reply by andrei-tatar** [reply]
@alonsomarce can you provide more details on what steps you took and what error did you get?
Do you have any devices created in node-red and synced with Smart NORA?
